doc/changelog/10-standard-library/14087-q-lemmas/qlemmas.rst

```diff
@@ -0,0 +1,16 @@
+- **Added:**
+  Lemmas to ``Q``: ``Qeq_from_parts``, ``Qden_cancel``, ``Qnum_cancel`` in theories/QArith/QArith_base.v.
+  Establishing equality of rationals by proving equality of numerator/denominator separately.
+  (`#14087 <https://github.com/coq/coq/pull/14087>`_,
+  by Karolin Varner).
```

Comment:
Generally there is only one changelog entry for each PR (not 3), usually with a single sentence summary.  See `doc/changelog` for examples.  The changelog is not meant to be documentation.

Also, it's probably enough just to mention `ZArith` rather than give pathnames.

Reply:
I've squashed and shortened the changelog entry.

If there are more small-ish changes like these – and there may well be some as I am obviously not yet acquainted with coding practices in the coq std lib – I would suggest that you just apply them while merging as this would probably faster for both of us compared to this back and forth.
If you do that please notify me so I can compare and see how to improve my PR next time!

Thanks for the timely reviews :pray:

Reply:
It looks good to me now, thanks.

Generally reviewers don't modify PRs for IMHO a couple reasons:
- Reviewers are a scarce resource (as are submitters!).  All reviewers are volunteers who have many demands on their time (even the Inria researchers who mostly work on Coq).
- Extra overhead to pull the PR code onto the reviewer's system and later push to GitHub again.  If a reviewer does that and the submitter makes subsequent changes, the submitter needs to pull the reviewer's changes.  This is prone to user errors that lose changes (i.e. forgetting to pull such changes).
- We want all changes to have independent review.  If the first reviewer makes significant changes, then we expect another reviewer will give the final approval.

Specific reviewers are expected based on what code is changed.  I mostly review documentation.  We need an "asignee" to give final approval and merge the code.  @herbelin, are you available to do that?  (or @Zimmi48 do you have another suggestion?)

Reply:
PS I unresolved the conversation so it's not hidden.

Reply:
@jfehrle Sometimes it is indeed more efficient for reviewers to push changes directly (avoids a lot of back and forth). If the changes are trivial, this is fine. If not, then the reviewer becomes a co-author and cannot be the assignee. GitHub tooling can help to make checking out a PR locally a breeze (see the "open with" button at the top-left of the PR). Pushing is a bit more complex: you need to use `git push FORK_URL`. And it is true that the submitter needs to remember pulling the reviewer's changes (the same is true with the use of inline suggestions during reviews though).